test/functional/terminal/tui_spec.lua

```diff
@@ -3554,26 +3643,7 @@ describe('TUI client', function()
     set_session(server_super)
     feed_data(status and ':' .. status .. 'cquit!\n' or ':quit!\n')
     status = status and status or 0
-    screen_server:expect {
-      grid = [[
-                                                        |
-      [Process exited ]] .. status .. [[]^ {MATCH:%s+}|
-                                                        |*4
-      {3:-- TERMINAL --}                                    |
-    ]],
-    }
-    -- assert that client has exited
-    screen_client:expect {
```

Comment:
@zeertzjq this expect() is the same as the one above. Is that intentional?

Reply:
It's not the same. The session is different.

Reply:
Ok restored, but see https://github.com/neovim/neovim/pull/32385#discussion_r1948274411